GOVERNANCE.md

```diff
@@ -0,0 +1,41 @@
+# Governance Policy
+
+This document provides the governance policy for the Project. Maintainers agree to this policy and to abide by all Project policies, including the [code of conduct](./CODE-OF-CONDUCT.md) by adding their name to the [`osrd.yaml` file](https://github.com/OpenRailAssociation/openrail-org-config/blob/main/teams/osrd.yaml).
+
+## 1. Roles.
+
+This project may include the following roles. Additional roles may be adopted and documented by the Project.
+
+**1.1. Maintainers**. Maintainers are responsible for organizing activities around developing, maintaining, and updating the Project. Maintainers are also responsible for determining consensus. This Project may add or remove Maintainers with the approval of the current Maintainers.
+
+**1.2. Contributors**. Contributors are those that have made contributions to the Project.
+
+## 2. Decisions.
+
+**2.1. Consensus-Based Decision Making**. Projects make decisions through consensus of the Maintainers. While explicit agreement of all Maintainers is preferred, it is not required for consensus. Rather, the Maintainers will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the Contributors and nature of support and objections. The Maintainers will document evidence of consensus in accordance with these requirements.
+
+**2.2. Appeal Process**. Decisions may be appealed by opening an issue and that appeal will be considered by the Maintainers in good faith, who will respond in writing within a reasonable time. If the Maintainers deny the appeal, the appeal may be brought before project admins, who will also respond in writing in a reasonable time.
+
+## 3. How We Work.
+
+**3.1. Openness**. Participation is open to anyone who is directly and materially affected by the activity in question. There shall be no undue financial barriers to participation.
+
+**3.2. Balance**. The development process should balance the interests of Contributors and other stakeholders. Contributors from diverse interest categories shall be sought with the objective of achieving balance.
+
+**3.3. Coordination and Harmonization**. Good faith efforts shall be made to resolve potential conflicts or incompatibility between releases in this Project.
+
+**3.4. Consideration of Views and Objections**. Prompt consideration shall be given to the written views and objections of all Contributors.
+
+**3.5. Written procedures**. This governance document and other materials documenting this project's development process shall be available to any interested person.
+
+## 4. No Confidentiality.
+
+Information disclosed in connection with any Project activity, including but not limited to meetings, contributions, and submissions, is not confidential, regardless of any markings or statements to the contrary.
+
+## 5. Amendments.
+
+Amendments to this governance policy may be made by affirmative vote of 2/3 of all Maintainers, with approval by project admins.
```

Comment:
just a little question:
why only 2/3 of all...,
why not all Maintainers (or e.g., 83%, 89%, or 97%) ?

Reply:
This comes from MVG: https://github.com/github/MVG/blob/main/project-docs/GOVERNANCE.md#6-amendments

Reply:
It comes from the MVG [here](https://github.com/github/MVG/blob/main/project-docs/GOVERNANCE.md). The idea is to allow modifications a small group of maintainers agree without having to check with everyone.

Reply:
Ok. Do not know how important amendments or governance is, but the idea is:
for the first version of this governance file, we could use **97% of all Maintainers**,
and later (as needed) if all Maintainers agree, we could reduce the rate to **67%** (2/3), or e.g.,
71%, 73%, 79%, 83%,...

```suggestion
Amendments to this governance policy may be made by affirmative vote of 97% of all Maintainers, with approval by project admins.
```

Reply:
I don't think that's a critical point. I prefer to have a static reasonable percentage rather than changing it when needed.